dev-tools/scripts/smokeTestRelease.py

```diff
@@ -14,35 +16,31 @@
 # limitations under the License.
 
 import argparse
-import os
```

Comment:
Mostly auto rearranged imports. I could leave them as they were but this is more organized :)